<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QDialog" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="Txt_Arquivo_Entrada">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arquivo Entrada</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Btn_Select_Arquivo_Entrada">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arquivo Saída</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Btn_Select_Arquivo_Saida">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>90</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_Arquivo_Saida">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Demandas</string>
   </property>
   <widget class="QPushButton" name="Btn_Demandas">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>22</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Calcula USTs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Calcula_SLAs">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>62</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Calcula SLA's</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Analisa Arquivos TXT</string>
   </property>
   <widget class="QPushButton" name="Btn_Analisa_NETSHARE">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Analisa 'NET SHARE'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Analisa_Desc">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>20</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Analisa DESC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Extrai_Areas_Report">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>60</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Extrai Áreas (Report)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Analisa_Dir_Gera_Comandos">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Analisa &quot;dir&quot; gera cmds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Analisa_Desc_FLevel_DbSchema">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>100</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Analisa DESC (FLevel)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Analisa_Dir_Caso_Desenv">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Analisa &quot;dir&quot; - Caso Desenv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Tamanho_VOBs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Tamanho VOBs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Analisa_LSVOB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Analisa LSVOB</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>321</y>
     <width>751</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Comunidades</string>
   </property>
   <widget class="QListWidget" name="Lista_Comunidades">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>130</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>RDNG</string>
   </property>
   <widget class="QPushButton" name="Btn_RDNG_Remocao_Links">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>21</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Remoção de links</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_RDNG_Gerar_Lista_Areas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Gerar lista de areas</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
